book/src/external-attacher.md

```diff
@@ -8,6 +8,7 @@
 
 Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | Min K8s Version | Max K8s Version | Recommended K8s Version |
 --|--|--|--|--|--|--|--
+[external-attacher v1.2.0](https://github.com/kubernetes-csi/external-attacher/releases/tag/v1.2.0) | [release-1.2](https://github.com/kubernetes-csi/external-attacher/tree/release-1.2) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | quay.io/k8scsi/csi-attacher:v1.2.0 | v1.13 | - | v1.15
```

Comment:
Maybe we should also add a column "New functionality"? i.e. what is new in 1.1 release what is new in the 1.2 release...

Reply:
Each repo has CHANGELOG files with all the new features and bug fixes